turbinia/evidence.py

```diff
@@ -356,9 +372,16 @@ def __init__(self, project=None, zone=None, disk_name=None, *args, **kwargs):
     self.cloud_only = True
 
   def _preprocess(self):
-    self.device_path = google_cloud.PreprocessAttachDisk(self.disk_name)
-    self.mount_path = mount_local.PreprocessMountDisk(
-        self.device_path, self.mount_partition)
+    self.device_path, partition_paths = google_cloud.PreprocessAttachDisk(
+        self.disk_name)
+    if self.mount_partition > len(partition_paths):
+      raise TurbiniaException(
+          ('Can not mount partition {0:d}: found only {1:d} partitions in '
+          'Google Cloud Disk {0:s}').format(
+              self.mount_partition, len(partition_paths), self.disk_name))
+
+    partition_path = partition_paths[self.mount_partition]
```

Comment:
This will actually return the second partition, not the first.  I think there are other instances of this as well.

Reply:
Good catch, tracking those down and adding a comment.